core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.spec.ts

```diff
@@ -54,7 +54,7 @@ describe('VisualizationPanelContentComponent', () => {
     spyOn(workflowStatusService, 'getCurrentResult').and.returnValue(testData);
 
     component.operatorID = 'operator1';
-    component.ngAfterViewInit();
+    component.ngAfterContentInit();
```

Comment:
why are we changing other test cases?

Reply:
because the function no longer there it is now renamed